sass/px-alert-message.scss

```diff
@@ -62,25 +62,46 @@ $actionable: true;
   --px-alert-label-text-color--information: black;
   --px-alert-label-background-color--unknown: white;
   --px-alert-label-text-color--unknown: black;
+  --px-btn-min-width: 0
 }
+[hidden] {
+  visibility: hidden;
+  display: none;
+}
+
 .alert-message {
+  visibility: hidden;
+  display: none;
   width: $inuit-base-spacing-unit--small * 40;
   margin: $inuit-base-spacing-unit--tiny;
   padding: $inuit-base-spacing-unit--small;
   min-height: $inuit-base-spacing-unit--small * 6;
-  animation: fadein .4s cubic-bezier(.78,.13,.16,.87);
   word-wrap: break-word;
   word-break: break-word;
   background-color: var(--px-alert-message-background-color, white);
-  opacity: 1;
   position: relative;
+  min-height: 0px;
```

Comment:
👍  we decided to stick with min-height because the browser is better at animating it